.github/workflows/github-ci.yml

```diff
@@ -86,6 +86,9 @@ jobs:
     - name: Install dependencies
       run: npm ci
 
+    - run: npm ls --prod
```

Comment:
I think this check should rather be done within the `test-matrix`. In here we just use different karma versions but not different node/npm versions.

Reply:
Right! Fixed